ui/app/mainui/AppMain.qml

```diff
@@ -1244,10 +1244,14 @@ Item {
 
                     currentIndex: {
                         const activeSectionType = appMain.rootStore.mainModuleInst.activeSection.sectionType
```

Comment:
Since the section change comes from Nim itself, I think you could hook into the `sectionType` change proc or something to send the event.

Reply:
Yes, I think I will send metrics from Nim layer wherever it is possible.